app/routes/task.py

```diff
@@ -0,0 +1,109 @@
+from flask import Blueprint, jsonify, request
+from app import db
+from datetime import date
+import requests
+import os
+
+from app.models.task import Task
+from app.routes.routes_helper import get_valid_item_by_id
+
+
+SLACK_ENDPOINT = "https://slack.com/api/chat.postMessage"
+SLACK_OATH_TOKEN = os.environ.get('SLACK_BOT_USER_OATH_TOKEN')
+SLACK_HEADERS = {
+    "Authorization": f"Bearer {SLACK_OATH_TOKEN}"
+}
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+def send_slack_msg(new_data):
+    slack_response = requests.post(SLACK_ENDPOINT, headers=SLACK_HEADERS, json=new_data)
```

Comment:
What I like about you turning this into its own separate helper method is that it helps split the responsibilities of your code well:

The route method (`mark_task` route) gets to focus on its one responsibility: Creating a response

This separate method `send_slack_msg` has all of the details about how to send a Slack message. If there was ever a time to say "Hey let's pull out that logic into its own route/class/model/etc," it'd be very easy to know where it was defined and modify it.

Well done!